sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/SpannerWriteIT.java

```diff
@@ -149,7 +149,9 @@ public void testWrite() throws Exception {
 
   @After
   public void tearDown() throws Exception {
-    databaseAdminClient.dropDatabase(options.getInstanceId(), databaseName);
+    if (databaseAdminClient != null) {
```

Comment:
Yea. And actually I think that this should never be null and it is caused by a broken string. So I think #3459 is for the NPE and we should finish solving the dependencies on this PR.